app/parse.py

```diff
@@ -7,9 +17,61 @@ class Quote:
     author: str
     tags: list[str]
 
+QUOTES_FIELDS = [field.name for field in fields(Quote)]
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)8s]: %(message)s",
+    handlers=[
+        logging.FileHandler(os.path.join("parser.log")),
+        logging.StreamHandler(sys.stdout),
+    ],
+)
+
+def parse_single_quote(quote) -> Quote:
+    tags_soup = quote.select_one(".tags")
+    tags = [tag.text for tag in tags_soup.select(".tag")]
+    return Quote(
+        text=quote.select_one(".text").text,
+        author=quote.select_one(".author").text,
+        tags=tags
+    )
+
+def get_single_page_quotes(page_soup: BeautifulSoup) -> [Quote]:
```

Comment:
The return type hint for `get_single_page_quotes` should be `list[Quote]` instead of `[Quote]`. The current hint suggests a list containing a single `Quote` object, which is misleading.